metadata-ingestion/source_docs/tableau.md

```diff
@@ -414,13 +462,15 @@ source:
     default_schema_map:
       mydatabase: public
       anotherdatabase: anotherschema
-    
+
```

Comment:
Please change `password: pass` to use environment variable instead like in Snowflake Source https://datahubproject.io/docs/metadata-ingestion/source_docs/snowflake/. We have seen people using secrets in plaintext sometimes. We would like to discourage that by having our examples have env variables instead

Reply:
Done